lib/phlex/rails/context.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+ApplicationControllerForPhlex = Class.new(ActionController::Base)
```

Comment:
It’s necessary to have a controller at the root namespace otherwise ActionView searches for partials inside the folder that matches that namespace of the controller.